src/drag_race_iarrc/include/LidarObstacle.h

```diff
@@ -0,0 +1,245 @@
+/*
+ * Created By: Gareth Ellis
+ * Created On: January 22, 2017
+ * Description: This class represents an Obstacle, as detected by the lidar,
+ *              and provides various utilities for working with obstacles
+ */
+
+#ifndef LIDAROBSTACLE_H
```

Comment:
Convention for this line in each of the files is all over the place. Some have the package name included and some don't. Not really sure what the proper convention is though haha, but if the package name is included, it should now be DRAG_RACE_IARRC_xxxxx_H.

Reply:
Hmm, that's a good point, doesn't look like we've done a very good job at keeping this consistent. :sweat_smile: In this case I think your idea here seems pretty good. It matches up with what we're doing in the sample package, and is unique enough that we are unlikely to have conflicts (with multiple headers having the same header guards).